### DISCUSSION TITLE
Graphic audio?

### DISCUSSION THREAD

**Discussion by Neurrone** [seanap/Plex-Audiobook-Guide#61] [Q&A] [answered]
How would you organize graphic audio books? Since they aren't on Audible. I guess the Plex agent won't work for those as well?

**Comment by seanap** [comment, marked as answer]
Some of them are on audible, here is an example of how I tagged one below.  The plex agent won't match books not in audible for now, but the dev is working on audiobookDB.org (abdb). The hope is that db will include all of audible and include other sources.  The plex agent will then point to abdb instead of audible, but that is still a wip.

![image](https://user-images.githubusercontent.com/17012946/155021892-69d75987-2e57-43f3-9629-d5b0df719f0b.png)

**Reply by Neurrone** [reply]
Thanks. So if the agent isn't able to match, then Plex falls back to using the ID3 tags and hence, it will be visible in Prologue?

I'm also wondering if you know how the developer for audiobookdb.org is getting the graphic audio data? Is there an API much like what Audible has?

**Reply by seanap** [reply]
That's right, Plex falls bak to using the _limited_ ID3 tags it can import, you should get ALBUM ALBUMARTIST ARTIST SORTALBUM YEAR DESC (if m4b) and the cover.  Missing: Rating, Publisher, Summary (if mp3), Genre's, Narrator (Style), and Series (Mood).

I don't believe any work has started on other sources (like storytel, graphicauidio, etc.) it is a massive undertaking.  If no api is available then he will probably seed the db by simple web scraping.

**Comment by binyaminyblatt** [comment]
i scraped the entire Graphic audio site 
[index.js](https://github.com/user-attachments/files/22323186/index.js)
[results.json](https://github.com/user-attachments/files/22323187/results.json)
[direct-store-exclusives.json](https://github.com/user-attachments/files/22323188/direct-store-exclusives.json)

**Reply by binyaminyblatt** [reply]
https://github.com/binyaminyblatt/graphicaudio_scraper
